swap_meet/clothing.py

```diff
@@ -1,2 +1,22 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+# Clothing Class extends Item Class
+class Clothing(Item):
+    def __init__(self, id=None, fabric="Unknown", condition=0.0):
+        super().__init__(category="Clothing", condition=condition, id=id)
+        self.fabric = fabric
+        self.condition = condition
+
+    
+    # return f"An object of type Clothing with id {id_value}. It is made from {fabric_value} fabric."
+    def get_category(self):
+        # return self.__class__.__name__
+        return "Clothing"
+
+    # has a function get_category to returns Clothing
+    def __str__(self):
+        id = self.id
+        fabric = self.fabric
+        
+        result = f"An object of type Clothing with id {id}. It is made from {fabric} fabric."
```

Comment:
There isn't a need to create a variables `id` or `fabric` and have them reference the values of the attribute `self.id` and `self.fabric`. We can directly get the attribute in the string, like I've suggested below:

```suggestion
        
        result = f"An object of type Clothing with id {self.id}. It is made from {self.fabric} fabric."
```

Reply:
Notice that the string you create on line 21 looks very similar to the what you have in the `__str__` method in the parent class in `item.py`. 

Here on line 21 you have `f"An object of type Clothing with id {id}."` and on line 37 of `item.py` you have `f"An object of type {item} with id {id}."`

Why not call the parent `__str__` method and get some of the behavior from the parent class and avoid repetition here?

```py
    def __str__(self):
        type_and_id_message = super().__str__()
        return f"{type_and_id_message} It is made from {fabric} fabric."
```